app/server/utils/user.py

```diff
@@ -792,7 +792,7 @@ def default_token(user: User) -> Token:
     return token
 
 
-def get_user_by_phone(phone: str, region: str, should_raise=False) -> Optional[User]:
+def get_user_by_phone(phone: str, region: Optional[str] = None, should_raise=False) -> Optional[User]:
```

Comment:
Made region optional because the `proccess_phone_number` function it uses doesn't require the region-- it defaults on `master_organisation().country_code`

Reply:
That makes sense